src/app/AppLayout/AppLayout.tsx

```diff
@@ -329,20 +329,20 @@ const AppLayout: React.FunctionComponent<IAppLayout> = ({ children }) => {
         logo={
           <>
             <Brand alt="Cryostat" src={cryostatLogo} className="cryostat-logo" />
-            <FeatureFlag strict level={FeatureLevel.DEVELOPMENT}>
-              <PageHeaderToolsItem>
-                <Label style={{ marginLeft: '2ch' }} isCompact color="red">
-                  Development
-                </Label>
-              </PageHeaderToolsItem>
-            </FeatureFlag>
-            <FeatureFlag strict level={FeatureLevel.BETA}>
-              <PageHeaderToolsItem>
-                <Label style={{ marginLeft: '2ch' }} isCompact color="green">
-                  Beta
-                </Label>
-              </PageHeaderToolsItem>
-            </FeatureFlag>
+            <DynamicFeatureFlag
+              levels={[FeatureLevel.DEVELOPMENT, FeatureLevel.BETA]}
+              component={(level) => (
```

Comment:
Maybe this function `(level) => {}` should be "memo". Right now, React will treat this as a completely new component everything there is a rebuild as a new function is generated. Everything under will be unmounted.

Reply:
Tho we should still use `React.useMemo` to cache this function.